src/framework-configuration.js

```diff
@@ -132,13 +150,30 @@ export class FrameworkConfiguration {
   constructor(aurelia: Aurelia) {
     this.aurelia = aurelia;
     this.container = aurelia.container;
+    /**
+     * Plugin / feature loadind instruction
+     * @type {FrameworkPluginInfo[]}
+     */
     this.info = [];
     this.processed = false;
     this.preTasks = [];
     this.postTasks = [];
+    /**
+     * Custom element's metadata queue for loading view factory
+     * @type {HtmlBehaviorResource[]}
+     */
+    this.behaviorsToLoad = [];
+    this.queuedPlugins = [];
     this.resourcesToLoad = {};
     this.preTask(() => aurelia.loader.normalize('aurelia-bootstrapper').then(name => this.bootstrapperName = name));
-    this.postTask(() => loadResources(aurelia, this.resourcesToLoad, aurelia.resources));
+    this.postTask(() => {
+      // if devs want to go all in static, and remove loader
+      // the following code shouldn't run
+      // add a check to make sure it only runs when there is something to do so
+      if (Object.keys(this.resourcesToLoad).length) {
```

Comment:
Nitpicking but the mainline code would be cleaner if you moved that condition inside `loadResources`. It would keep concerns close together.